server/models/user.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const UserSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  date: {
```

Comment:
I believe this is meant to be createdAt date ? if so - i would prefer to name this as `createdAt` so that it's not descriptive